client/src/components/Navbar/Navbar.module.scss

```diff
@@ -1,3 +1,5 @@
+@import "../../index.css";
```

Comment:
I understand your justification, but I don't think this will be necessary once #126 is merged in. That issue should change the font family project-wide without having to do any extra work within the components themselves. If that's not the case, then you should change the `font-family` directly, not import the index file.

Reply:
@KingJordan152 I implemented #126 changes on my local environment and it still works without importing the `index.css`. I kept it in for now and will delete it once #126 is resolved. It just creates a duplicated `:root` attribute.

Reply:
The changes were made.